io.github.iptux_src.iptux.yml

```diff
@@ -0,0 +1,58 @@
+# snapcraft.yaml
+name: iptux
+version: 0.9.3
+app-id: io.github.iptux_src.iptux
+runtime: org.gnome.Platform
+runtime-version: '46'
```

Comment:
current is 47

Reply:
does it realluy need the GNOME runtime?

Reply:
it depends on GTK3, any other more proper option?

Reply:
freedesktop sdk has gtk3.